src/test/java/com/codurance/retropolis/applicationservices/ApplicationCardServiceTest.java

```diff
@@ -0,0 +1,120 @@
+package com.codurance.retropolis.applicationservices;
+
+import static java.util.Collections.emptyList;
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.mockito.Mockito.verify;
+import static org.mockito.Mockito.when;
+
+import com.codurance.retropolis.entities.Card;
+import com.codurance.retropolis.entities.User;
+import com.codurance.retropolis.services.CardService;
+import com.codurance.retropolis.services.UserService;
+import com.codurance.retropolis.web.requests.NewCardRequestObject;
+import com.codurance.retropolis.web.requests.UpVoteRequestObject;
+import com.codurance.retropolis.web.requests.UpdateCardRequestObject;
+import com.codurance.retropolis.web.responses.CardResponseObject;
+import com.codurance.retropolis.web.responses.CardResponseObjectFactory;
+import com.codurance.retropolis.web.responses.CardUpdatedTextResponseObject;
+import java.util.Collections;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.extension.ExtendWith;
+import org.mockito.Mock;
+import org.mockito.junit.jupiter.MockitoExtension;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+
+@ExtendWith(MockitoExtension.class)
+public class ApplicationCardServiceTest {
+
+  private final Boolean HAVE_VOTED = false;
+  private final String EMAIL = "john.doe@codurance.com";
+  private final User USER = new User(3L, "john.doe@codurance.com", "John Doe");
+  private final Long CARD_ID = 1L;
+  private final Long COLUMN_ID = 2L;
+  private final String TEXT = "Hello";
+  @Mock
+  UserService userService;
```

Comment:
Can you please make those fields private?

Reply:
Thanks. Added.